front/src/applications/rollingStockEditor/json/rollingStockEditorData.json

```
@@ -0,0 +1,139 @@
+{
+  "name": {
+    "title": "name",
+    "type": "string",
+    "side": "left"
+  },
```

Transform your json file into a TS one :)

```suggestion
import { SchemaProperty } from '../consts';

const RollingStockEditorSchema: SchemaProperty[] = [
  {
    title: 'name',
    type: 'string',
    side: 'left',
  }, ...
];

export default RollingStockEditorSchema;

```

---

done